app/views/shared/_analytics_table.html.erb

```diff
@@ -73,37 +73,73 @@
         <% end %>
       </tr>
 
-      <% policy_scope([:cohort_report, row_resource]).each do |resource| %>
-        <% if dashboard_analytics[resource.id].present? %>
-          <tr>
-            <td class="row-title">
-              <% if row_resource_link == :analytics_facility_path %>
-                <%= link_to resource.send(row_resource_display_field),
-                            send(row_resource_link, resource, period: @period) %>
-              <% else %>
-                <%= link_to resource.send(row_resource_display_field), send(row_resource_link, resource) %>
-              <% end %>
-            </td>
-
-            <!--Registered patients-->
-            <td style="text-align: center;">
-              <%= dash_if_zero(dashboard_analytics.dig(resource.id, :total_registered_patients)) %>
-            </td>
+      <% if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) %>
```

Comment:
Can we simplify these big chunks of ERB branching by pulling out just the bits we need to switch on? In both cases we're getting some collection in two different ways, but iterating through it the same way.

Can we use the Flipper if-else just to set a local `resources` var, and then have a single `resource.each` block?

Reply:
Will do this.

I've optimized for easily deleting the `else` branches, but it does lead to big mostly-duplicated chunks.

Reply:
This is done